mlir/test/e2e/gemm_with_atomic_add.toml

```diff
@@ -9,7 +9,7 @@ prefix = "--store-method="
 
 [[axis]]
 name = "data type"
-values = ["f32", "f16"]
+values = ["f32", "f16 -RMS_threshold=1e-3"]
```

Comment:
I thought we are not supporting f16

Reply:
Currently, you can use `GEMMs` with `fp16` and the `atomic-add` as the store method. The split-k GEMM doesn't allow to do it. I hope that @giuseros has already figured out what was wrong and the fix is coming. This test doesn't fail. I would leave it because I am afraid that I would forget to add it again in the future.